Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -194,6 +194,10 @@ void GLContextWidget::mousewheel_event(GUI::MouseEvent& event)
 
 void GLContextWidget::keydown_event(GUI::KeyEvent& event)
 {
+    window()->propagate_shortcuts_up_to_application(event, this);
```

Comment:
Instead of calling shortcuts before running the event, you should call `event.ignore()` at the end of this function. Calling that function allows the event to bubble up to the Window, where shortcuts are handled. When keydown_event is called, `event.is_accepted()` is true, so the `return;` in the Escape check below will prevent `event.ignore()` from being called. Thus, the event will only bubble up and turn into a shortcut activation if this function doesn't do anything with the event and return early.

Reply:
Thanks for the help. I'm not too familiar with the system so I wasn't aware of that. Let me know if the changes I made are good.

Reply:
Looks right to me! If it fixes the issue, then this should be good to go from my perspective.